pyquil/noise.py

```diff
@@ -296,22 +296,51 @@ def damping_after_dephasing(T1, T2, gate_time):
 
 # You can only apply gate-noise to non-parametrized gates or parametrized gates at fixed parameters.
 NO_NOISE = ["RZ"]
-NOISY_GATES = {
-    ("I", ()): (np.eye(2), "NOISY-I"),
-    ("RX", (np.pi / 2,)): (np.array([[1, -1j],
-                                     [-1j, 1]]) / np.sqrt(2),
-                           "NOISY-RX-PLUS-90"),
-    ("RX", (-np.pi / 2,)): (np.array([[1, 1j],
-                                      [1j, 1]]) / np.sqrt(2),
-                            "NOISY-RX-MINUS-90"),
-    ("RX", (np.pi,)): (np.array([[0, -1j],
-                                 [-1j, 0]]),
-                       "NOISY-RX-PLUS-180"),
-    ("RX", (-np.pi,)): (np.array([[0, 1j],
-                                  [1j, 0]]),
-                        "NOISY-RX-MINUS-180"),
-    ("CZ", ()): (np.diag([1, 1, 1, -1]), "NOISY-CZ"),
-}
+ANGLE_TOLERANCE = 1e-10
```

Comment:
My only concern is that this may be too forgiving. Not sure tho...

Reply:
We can make this stricter once we reach ~10e-10 gate error :)

Reply:
good point!